src/spaceone/dashboard/model/custom_widget_model.py

```diff
@@ -39,6 +41,7 @@ class CustomWidget(MongoModel):
             'widget_name',
             'title',
             'version',
+            'settings',
```

Comment:
Object type are not efficient even when indexed.
Please remove the `settings` index.